tests/macros/utils.test.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @prettier
```

Comment:
Thank you for adding this test.

Since you are adding the @prettier comments to your tests, did you remember to `npm run format` before submitting the PR? (I don't see anything that looks incorrectly formatted... just asking because I forget to do it myself.)

Reply:
I run <code>[pnpx](https://pnpm.js.org) prettier <path&#x2011;to&#x2011;file> &#x2011;&#x2011;write</code> manually.

Reply:
If you use the 'format' script in package.json, it will ensure that only files with the @prettier comment get formatted. It doesn't matter how you run it, though, as long as you're using whatever version of prettier is installed in package.json, and not a global install.